internal/xds/translator/testdata/in/xds-ir/accesslog-als-tcp.yaml

```diff
@@ -7,6 +7,7 @@ accesslog:
       path: "%REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%"
       protocol: "%PROTOCOL%"
       response_code: "%RESPONSE_CODE%"
+    logType: Global
```

Comment:
can we avoid this? that will make this PR a little smaller?

Reply:
done